_docs_integrate/version-5.md

```diff
@@ -99,6 +99,60 @@ In addition, in contrast to the RequestItems, a RequestItemGroup did not have to
+
+The `content` of a [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate) must either be a [RelationshipTemplateContent]({% link _docs_integrate/data-model-overview.md %}#relationshipcreationcontent) or an [ArbitraryRelationshipTemplateContent]({% link _docs_integrate/data-model-overview.md %}#arbitraryrelationshiptemplatecontent).
+
+The `creationContent` of a [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) must either be a [RelationshipCreationContent]({% link _docs_integrate/data-model-overview.md %}#relationshipcreationcontent) or an [ArbitraryRelationshipCreationContent]({% link _docs_integrate/data-model-overview.md %}#arbitraryrelationshipcreationcontent).
+
+Those new wrappers all have the type
+
+```jsonc
+{
+  "@type": "Arbitrary<...>Content",
+  "value": any,
+}
+```
+
+Thus, when sending a Message with non-standard content via POST `Messages`, instead of sending the parameter `content : <arbitrary-content>`, you send
+
+```jsonc
+content: {
+  "@type": "ArbitraryMessageContent",
+  "value": <arbitrary-content>
+}
+```
+
+Similarly, when creating a RelationshipTemplate with non-standard content via POST `RelationshipTemplates`, you send the parameter
+
+```jsonc
+content: {
+  "@type": "ArbitraryRelationshipTemplateContent",
+  "value": <arbitrary-content>
+}
+```
+
+and the parameter
+
+```jsonc
+creationContent: {
+  "@type": "ArbitraryRelationshipCreationContent",
+  "value": <arbitrary-content>
+}
+```
+
+for creating a Relationship via POST `Relationship`.
+
+When reading the value from the RelationshipTemplate resp. Message, the relevant property is thus changed from `content` to `content.value`.
+
+## Backbone-Sync returns 204 (No content)
+
+Previosly, the Backbone-Sync (via the route POST `Account/Sync`) returned a list of Relationships and Messages with the HTTP code 201. This has been changed to the sync returning nothing with the HTTP code 204. This removes the support for the workflow of regularly calling the sync and checking the response since unexpected response values happen if syncs are executed from different spots. We recommend working event-based with the [Message Broker Publisher-Module]({% link _docs_operate/modules.md %}#messagebrokerpublisher), which sends events to a message broker you have set up. We send a variety of events like `incomingRequestReceived`, `incomingRequestStatusChanged`, each of them containing the relevant changed object. See [Connector Events] for the full list. Slightly related, we also recommend using the recently published [server-sent-events Module]{% link _docs_operate/modules.md %}#sse, which listens to events emitted from the Backbone and syncs with the Backbone when an event is received.
```

Comment:
Fix the link "[server-sent-events Module]{% link _docs_operate/modules.md %}#sse" please, brackets are missing there.